wpilibNewCommands/src/main/native/cpp/frc2/command/Command.cpp

```diff
@@ -185,6 +186,31 @@ std::optional<std::string> Command::GetPreviousCompositionSite() const {
   return m_previousComposition;
 }
 
+void Command::EnsureDisjointRequirements(Command* toAdd) {
+  std::string sharedRequirementsStr = "";
+  bool hasSharedRequirements = false;
+  auto&& requirementsToAdd = toAdd->GetRequirements();
+  for (auto&& requirement : this->GetRequirements()) {
+    if (requirementsToAdd.find(requirement) != requirementsToAdd.end()) {
+      sharedRequirementsStr.append(requirement->GetName());
+      if (!hasSharedRequirements) {
+        hasSharedRequirements = true; // ensures formatting like "a, b, c"
+      } else {
+        sharedRequirementsStr.append(", ");
+      }
```

Comment:
```suggestion
      if (!hasSharedRequirements) {
        hasSharedRequirements = true; // ensures formatting like "a, b, c"
      } else {
        sharedRequirementsStr.append(", ");
      }
      sharedRequirementsStr.append(requirement->GetName());
```
Otherwise, shared requirements `[a, b, c]` would output `ab, c, `.

Reply:
Yeah i forgot to correct the order; thanks